src/MultiBridgeMessageSender.sol

```diff
@@ -281,7 +281,12 @@ contract MultiBridgeMessageSender {
             block.timestamp + _args.expiration
         );
         bytes32 msgId = MessageLibrary.computeMsgId(message);
-        bool[] memory adapterSuccess = _dispatchMessages(adapters, mmaReceiver, _args.dstChainId, message, _args.fees);
+        (bool[] memory adapterSuccess, uint256 successCount) =
+            _dispatchMessages(adapters, mmaReceiver, _args.dstChainId, message, _args.fees);
+
+        if (successCount == 0) {
```

Comment:
I don’t think that reverting only when all adapters fail is sufficient. We should instead consider whether enough bridges were successful in sending a message to achieve a quorum on the remote chain. If this is not the case, then the dispatching has ultimately failed, and we should revert the transaction to enable a retry. Otherwise, a new governance proposal will be necessary, which is not desirable.

Since we do not store the quorum threshold of remote chains on the source chain, I propose that we instead require the caller of remoteCall() to provide a success threshold value. If at least that number of bridges succeed, the dispatching will be considered successful. If not, it should revert.